universal-login-commons/lib/integration/ethereum/TokenDetailsService.ts

```diff
@@ -9,15 +10,21 @@ const tokenAbi = [
 export class TokenDetailsService {
   constructor(private provider: providers.Provider) {}
 
-  async getSymbol(tokenAddress: string) {
+  async getTokenDetails(tokenAddress: string): Promise<TokenDetails> {
+    const symbol = await this.getSymbol(tokenAddress);
+    const name = await this.getName(tokenAddress);
```

Comment:
I am not sure if we use name or symbol separately, so I would inline `getSymbol` and `getName`.